Vagrantfile

```diff
@@ -0,0 +1,78 @@
+# -*- mode: ruby -*-
```

Comment:
Any chance the Vagrantfile can sit in the contrib folder? 🙂

Reply:
sure thing